west.yml

```diff
@@ -242,7 +242,7 @@ manifest:
           upstream-sha: c6eaeda5a1c1c5dbb24dce7e027340cb8893a77b
           compare-by-default: false
     - name: suit-generator
-      revision: f07d308f03e039bc73767c145f4cff4a70e74442
+      revision: c0a59a898d0f8ce96937fa5adf4ffa4bb344a837
```

Comment:
Shouldn't this point to https://github.com/nrfconnect/suit-generator/commit/7cde03f9d87534f1eebfaae1cda5715d4299990d, the commit from main? This PR targets `main`, not `v2.9-nRF54H20-branch`.